swarm/pss/pss.go

```diff
@@ -303,102 +272,123 @@ func (self *Pss) checkFwdCache(addr []byte, digest pssDigest) bool {
 	return false
 }
 
+// DPA storage handler for message cache
+func (self *Pss) storeMsg(msg *PssMsg) (pssDigest, error) {
+	swg := &sync.WaitGroup{}
+	wwg := &sync.WaitGroup{}
```

Comment:
It is typical to call `Wait` on a `sync.WaitGroup`, is that not necessary here?

Reply:
To be perfectly honest, I don't 100% know what these two Waitgroups in the DPA do, and if I need to wait for both. Do you know?

Reply:
This wait group API is in desperate need of a rethink, I also find it very confusing.

@zelig can you clarify whether we need to wait on these wait groups here?